Source/WebCore/svg/SVGAnimateMotionElement.cpp

```diff
@@ -240,6 +233,16 @@ void SVGAnimateMotionElement::calculateAnimatedValue(float percentage, unsigned
         for (unsigned i = 0; i < repeatCount; ++i)
             buildTransformForProgress(transform, 1);
     }
+    float positionOnPath = m_animationPath.length() * percentage;
+    auto traversalState(m_animationPath.traversalStateAtLength(positionOnPath));
+
+    float angle = traversalState.normalAngle();
```

Comment:
Not obvious to the reader that `angle` is in degrees, but that's not new.

Reply:
Should I add comment or rename (e.g., 'degreeAngle') and then just do 'unsafe-merge' or it is to leave as is?

Reply:
`// The 'angle' below is in 'degrees'.` <- Added this comment. @smfr